indexing-hadoop/src/main/java/io/druid/indexer/InputRowSerde.java

```diff
@@ -233,7 +290,10 @@ public Double deserialize(ByteArrayDataInput in)
             typeHelper = STRING_HELPER;
           }
           writeString(dim, out);
-          typeHelper.serialize(out, row.getRaw(dim), reportParseExceptions);
+          String parseExceptionMessage = typeHelper.serialize(out, row.getRaw(dim), true);
```

Comment:
Probably `reportParseExceptions` should be passed instead of `true`?

Reply:
Removed `reportParseExceptions` here, it should always be true (always thrown, handling depends on config)